src/pages/organisationUnits/list/OrganisationUnitList.tsx

```diff
@@ -192,47 +193,60 @@ export const OrganisationUnitList = () => {
         state: {
             expanded,
         },
+        enableSubRowSelection: false,
     })
 
     return (
         <div>
             <SectionListTitle />
             <IdentifiableFilter />
-            <ToolbarNormal />
-            <SectionList
-                allSelected={table.getIsAllRowsSelected()}
-                headerColumns={table.getHeaderGroups()[0].headers.map((h) => ({
-                    label: h.column.columnDef.header as string,
-                    path: h.column.id,
-                }))}
-                onSelectAll={() => table.toggleAllRowsSelected()}
-            >
-                <OrganisationUnitListMessage
-                    isFiltering={isFiltering}
-                    queries={queries.concat(orgUnitFiltered)}
-                    orgUnitCount={table.getRowCount()}
+            <div className={css.listDetailsWrapper}>
```

Comment:
I guess we should add the details-panel soon rather than later as well.

Reply:
Good point. Created a story here and added it to the current sprint. I can work on it next